p2p/utils/logging.go

```diff
@@ -41,6 +54,63 @@ func LogIncomingMsg(logger *modules.Logger, hostname string, peer types.Peer) {
 	logMessage(logger, msg, hostname, peer)
 }
 
+// Print table prints a table to stdout. Header row is defined by `header`. Row printing
+// behavior is defined by `consumeRows`. Header length SHOULD match row length.
```

Comment:
I think something like this is a really good opportunity to just import another library: https://github.com/jedib0t/go-pretty?